<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>内窥镜息肉检测</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../.designer/backup/colonoscopy.png</normaloff>../../../.designer/backup/colonoscopy.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>991</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>单张图片检测</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>931</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="InputImgLabel">
         <property name="text">
          <string>输入图片路径：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="InputImgLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="OutputImgLabel">
         <property name="text">
          <string>输出图片路径：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="OutputImgLineEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="InputImgPushButton">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="OutputImgPushButton">
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ImageBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>931</width>
        <height>791</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ImageRecognize">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>920</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>识别</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>视频实时检测</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>931</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="InputVideoLabel">
         <property name="text">
          <string>输入视频路径：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="InputVideoLineEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="IntputVideoPushButton">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="OutputVideoLabel">
         <property name="text">
          <string>输出视频路径：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="OutputVideoLineEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="OutputVideoPushButton">
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="VideoRecognize">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>实时检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VideoRecognize_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>视频检测</string>
      </property>
     </widget>
     <widget class="QLabel" name="ImageBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>931</width>
        <height>791</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="VideoRecognize_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1010</y>
      <width>971</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
